### DISCUSSION TITLE
When using "export as PDF" some content does not appear or is cut off

### DISCUSSION THREAD

**Discussion by sierratull** [GitbookIO/community#90] [5. Export]
### What's your feature/idea?

The "export as PDF" option does not display the cover images for the Card content block and cuts off wide Table content blocks on laptop and mobile devices

### What's your desired outcome?

Have the ability to view all content (e.g. cover images, tables) fully

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by aikoii** [comment]
Hey @sierratull,

Our [Changelog ](https://changelog.gitbook.com/february-2023/07-february-23-monorepos-customized-commit-messages-and-more)would be the best place to view a list of all of the changes and fixes 🙂.

**Comment by taszap** [comment]
This issue is linked to #64

**Comment by ahcsco** [comment]
Is there any progress on this one? I'm surprised that it wasn't part of recent gitbook updates.
The current pdf export is pretty halfway i'd say, where a lot of gitbook specific formatting elements are either cut out or shows up funny, hindering us from using the feature. There's also not much custom options as linked in #64. 
Kindly prioritise this one.

**Reply by taszap** [reply]
Hi @shimshim938 thank you for your report. In general, we recommend using our sharing options (sharable links, visitor authentication etc) to share content with your readers. We have made some urgent fixes to PDFs this month and are looking at further improvements, but we are unlikely to provide more custom options or further develop it in the next few months.

**Reply by De-Ma** [reply]
Some of our customers need of grid access to the documentation to fix problems in the field. That's why sharable links/ visitor authentication,... Is not the full solution for us.

Some things have been fixed, but some content and headings are still split in half on page breaks, for example Headings and tables:
![More Pagebreak issues](https://github.com/GitbookIO/community/assets/75689014/605120a9-886f-415e-99b3-00d15b087c25)

We are also looking forward to some more features in PDF export, as linked in https://github.com/orgs/GitbookIO/discussions/64.

**Reply by JohnAtQNX** [reply]
Hiya! Just wanted to toss a note of support on this one. We too have customers who have a need to use product documentation in offline environments, such as in security/research labs where all access to outside networks is blocked. The PDFs are helpful for this purpose.
